src/canonicalBookMap/sociology.ts

```diff
@@ -5,199 +5,395 @@ import { CanonicalBookMap } from '../canonicalBookMap';
+      '7f6d4334-2d56-4995-88ea-be47c71c5733': '6e69fbe2-9c5d-4c03-85db-565c56621331',
+      /* Social Movements */
+      '385b5c8d-3fcd-4283-99fa-0312b514c05c': '1272acf1-44da-4f2c-88e7-4eff807738e6',
+      /* Social Change */
+      'b9fed02b-edda-433d-adfa-fce8cc55091d': '4937c948-e669-42a4-98cc-fe18f2370de1',
+    }],
+  ],
+  /* Introduction to Sociology 2e */ '02040312-72c8-441e-a685-20e9333f3e1d': [
+    /* Introduction to Sociology 3e */['746f171e-0d6a-4ef2-b69d-367880872f4a', {
+      /* Preface */
+      '325e4afd-80b6-44dd-87b6-35aff4f40eac': '6454d583-84a8-4ce1-9af2-36be28532b98',
+      /* 1.0 Introduction to Sociology */
+      '4eb21133-cf0a-4ea7-81f5-b1e47698ec30': 'bdf8b962-cd9d-4752-a983-cd3d57b50f02',
+      /* 1.1 What Is Sociology? */
+      '5d97ba77-626b-4670-a1bb-c65fdd07bb0c': 'c8d9ab4b-dd9c-4dad-aa9f-00bae82cd0af',
+      /* 1.2 The History of Sociology */
+      '82966abb-ebb0-48ab-8576-a52a0e126bd9': '5f36c280-1b4c-4abb-9407-61358242bee0',
+      /* 1.3 Theoretical Perspectives */
+      '40c45f23-6a75-414a-987a-cccd50bd04b8': '746efd2e-8fb4-4368-98b4-45f6fd33eb27',
+      /* 1.4 Why Study Sociology? */
+      '2478dd75-fc23-48c8-9dfc-57a409d04c6b': 'e13dda45-c995-47e5-8146-0ac7e1015ce6',
+      /* 2.0 Introduction to Sociological Research */
+      'de329f8c-7690-4ea5-a3a4-d0ce6605d0ea': '5f0b4a03-f79c-442a-9fae-0a8a1359f4df',
+      /* 2.1 Approaches to Sociological Research */
+      'baee4db6-1e28-43f9-a0b7-a4f3c7263598': '0f87b4b5-bdb9-478d-94e1-f6967392b249',
+      /* 2.2 Research Methods */
+      'e72e915a-719d-4496-ac15-5197a2c11258': '51969efe-44b2-4302-97b5-a50ed8bd0eaf',
+      /* 2.3 Ethical Concerns */
+      '6d922c72-61cf-4f13-9b00-4f0c640cbf17': '90ffbc39-5865-42ad-bcd9-bfb740dbdfd7',
+      /* 3.0 Introduction to Culture */
+      '8a8ed60b-fb85-463f-a002-b48af257f894': 'd54ed6a4-1098-4b2c-b206-b7850002fd26',
+      /* 3.1 What Is Culture? */
+      'f87398b0-553a-4538-959d-15b28d0a80ef': 'dde4d8cb-ce8f-4c68-bad2-786d7c0f5f29',
+      /* 3.2 Elements of Culture */
+      'f298104a-d0dd-432f-83ec-3881813a7eaa': '946e6080-b7be-495d-9679-667c0e985a18',
+      /* 3.3 Pop Culture, Subculture, and Cultural Change */
+      '4ee317f2-cc23-4075-b377-51ee4d11bb61': '5750bc50-77e4-42ed-a7a0-dcc31a562dcb',
+      /* 3.4 Theoretical Perspectives on Culture */
+      'f2055c29-6b50-4a77-89d8-f88745ce0f18': 'ca2347f6-ca5a-46cf-bd8b-75768bd084a3',
+      /* 4.0 Introduction to Society and Social interaction */
+      '31eae3d7-1b57-4c24-ad0b-d08a2851b70d': 'ef238876-14ce-464f-ac5a-e30d656d77b7',
+      /* 4.1 Types of Societies */
+      'aace8da1-b83e-4a5e-bfb8-5bd44699381a': '526f33fb-2ed8-4d15-b2e5-2b83a2b97849',
+      /* 4.2 Theoretical Perspectives on Society */
+      '6414e0c9-1e95-4b06-af38-b11762cb667e': 'c1bc1aa5-03ef-4909-90b3-f47d33bc0c85',
+      /* 4.3 Social Constructions of Reality */
+      'd06ad985-cecb-48b4-b222-197d04abc6a': '040ac094-d7eb-4761-840f-04f22aa30797',
```

Comment:
it should be `d06ad985-cecb-48b4-b222-197d04abc6a1`
there is a typo in the spreadsheet - missing `1` at the end and canonical for https://rex-web-issue-1595-vrsz9ftxs2e.herokuapp.com/books/introduction-sociology-2e/pages/4-3-social-constructions-of-reality is not being set